simplq/src/components/common/Footer.jsx

```diff
@@ -50,33 +50,33 @@ export default () => (
           <ul>
             <li>
               <a href="https://iimb.qualtrics.com/jfe/form/SV_aY8FY91ztRX9NvD">
-                <PhoneIcon fontSize={"large"} />
+                <PhoneIcon fontSize="large" />
               </a>
             </li>
             <li>
               <a href="https://github.com/SimplQ/simplQ-frontend">
-                <GitHubIcon fontSize={"large"} />
+                <GitHubIcon fontSize="large" />
               </a>
             </li>
             <li>
               <a href="https://medium.com/@raimazach/virtualizing-queues-a-long-overdue-alternative-to-physical-queues-bfdc4b51070f">
-               <img src="images/ICON-MEDIUM.jpg" />
+                <img alt="medium-link" src="/images/ICON-MEDIUM.jpg" />
               </a>
```

Comment:
Wasn't showing up on chrome because of the missing / at the beginning. Worked in firefox though. Strange.

Reply:
Strange